src/SystemManager.cc

```diff
@@ -34,11 +39,15 @@ SystemManager::SystemManager(const SystemLoaderPtr &_systemLoader,
   transport::NodeOptions opts;
   opts.SetNameSpace(_namespace);
   this->node = std::make_unique<transport::Node>(opts);
-  std::string entitySystemService{"entity/system/add"};
-  this->node->Advertise(entitySystemService,
+  std::string entitySystemAddService{"entity/system/add"};
+  this->node->Advertise(entitySystemAddService,
       &SystemManager::EntitySystemAddService, this);
   ignmsg << "Serving entity system service on ["
-         << "/" << entitySystemService << "]" << std::endl;
+         << "/" << entitySystemAddService << "]" << std::endl;
+
+  std::string entitySystemInfoService{"entity/system/info"};
```

Comment:
Having `entity` here is a bit confusing to me, because the service isn't returning anything related to a specific entity. How about `/gazebo/system/info`, `~/system/info` or something like that?

Reply:
changed to `~/system/info`. [74d1d0c](https://github.com/gazebosim/gz-sim/pull/1581/commits/74d1d0c904d43d73cac3843fbf0cd236c0070cb0)